geode-redis/src/distributedTest/java/org/apache/geode/redis/PubSubDUnitTest.java

```diff
@@ -46,13 +49,13 @@
   public static void beforeClass() {
     ports = AvailablePortHelper.getRandomAvailableTCPPorts(2);
     Properties redisProps = new Properties();
-    redisProps.put("redis-bind-address", "localhost");
+    redisProps.put(DistributionConfig.REDIS_BIND_ADDRESS_NAME, "localhost");
```

Comment:
Are these truly internal properties? Maybe there Redis module has it's own set of public properties.

Reply:
Well, they exist... There is currently no way, I'm aware of, for a module to expose its own set of parameters. It would be nice to add that.

Reply:
Firstly... the publicly accessible GemFire properties are under `ConfigurationProperties`... So adding them to an internal class is not really the answer.
Secondly, maybe as a Geode "extension" we need to add the ability to be able to extend our public properties list dependent on the extension. If that model does not exist, maybe it is safer to possibly not add the properties to an internal class (which the extension arguably should not have access to)..

Reply:
Switched to `ConfiguratonProperties`